client-frameworks-support/client-support-ui5/README.md

```diff
@@ -0,0 +1,2 @@
+## How to build
+npm run build
```

Comment:
Similarly to the [Client Support Angular Library](https://github.com/SAP/luigi/tree/main/client-frameworks-support/client-support-angular), the actual readme is "hidden" in a deeper folder. I'd suggest to add a note about it in the first-level readme so the user can find it. 

```suggestion
## Luigi Client UI5 Support Library
The Luigi Client UI5 support library provides several useful features to run your UI5 application inside the Luigi micro frontend framework.

You can find more information in the [main README file](https://github.com/SAP/luigi/tree/main/client-frameworks-support/client-support-ui5/src/README.md). 
```